Public/Src/Engine/Scheduler/Distribution/LocalWorker.cs

```diff
@@ -21,6 +23,13 @@ public sealed class LocalWorker : Worker
         /// </summary>
         public ConcurrentDictionary<PipId, Unit> CurrentlyExecutingPips = new ConcurrentDictionary<PipId, Unit>();
 
+        /// <summary>
+        /// The number of pips that are currently running (i.e., the associated pip process is still alive and running)
+        /// </summary>
+        public int CurrentlyRunningPipCount => m_currentlyRunningPipCount;

```

Comment:
>m_currentlyRunningPipCount [](start = 47, length = 26)

I think we need `Volatile.Read` for reading currently running pip count.

Reply:
will change